include/bluetooth/audio/lc3.h

```diff
@@ -183,23 +203,35 @@ struct bt_codec_lc3_frame_len {
 /** @def BT_CODEC_CONFIG_LC3_CHAN_ALLOC
  *  @brief LC3 channel Allocation configuration type
  */
-#define BT_CODEC_CONFIG_LC3_CHAN_ALLOC   (BT_CODEC_LC3_CONFIG_BASE + 2)
+#define BT_CODEC_CONFIG_LC3_CHAN_ALLOC   (0x03)
 
 /** @def BT_CODEC_CONFIG_LC3_FRAME_LEN
  *  @brief LC3 Frame Length configuration type
  */
-#define BT_CODEC_CONFIG_LC3_FRAME_LEN    (BT_CODEC_LC3_CONFIG_BASE + 3)
+#define BT_CODEC_CONFIG_LC3_FRAME_LEN    (0x04)
+
+
+/** @def BT_CODEC_CONFIG_LC3_BLKS_PER_SDU
+ *  @brief LC3 blocks per frame configuration type
+ */
+#define BT_CODEC_CONFIG_LC3_BLKS_PER_SDU (0x05)
```

Comment:
The brief says `blocks per frame` but the name is `blocks per SDU`. Is that correct?

Reply:
Yes, Thanks!